README.md

```diff
@@ -3,6 +3,65 @@
   <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
   <h1>🍕 Open Sauced Browser Extension(s) 🍕</h1>
 </div>
+<p align="center">
+  <img src="https://img.shields.io/github/languages/code-size/open-sauced/browser-extensions" alt="GitHub code size in bytes">
+  <img src="https://img.shields.io/github/commit-activity/w/open-sauced/browser-extensions" alt="GitHub commit activity">
+  <a href="https://github.com/open-sauced/browser-extensions/issues">
+    <img src="https://img.shields.io/github/issues/open-sauced/browser-extensions" alt="GitHub issues">
+  </a>
+  <a href="https://discord.gg/U2peSNf23P">
+    <img src="https://img.shields.io/discord/714698561081704529.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" alt="Discord">
+  </a>
+  <a href="https://twitter.com/saucedopen">
+    <img src="https://img.shields.io/twitter/follow/saucedopen?label=Follow&style=social" alt="Twitter">
+  </a>
+</p>
+
+## Goals
+To enable developers find 
+
+## Running the project locally
+
+To run the project, you'll need the following software binaries installed on your development machines:
+
+- [Node.js](https://nodejs.org/en)
+- [NPM](https://www.npmjs.com/)
+
+To install the project dependencies:
+
+```shell
+npm ci
+```
+
+To run a local instance of the project:
+
+```shell
+npm run dev
+```
+
+## Installing the extension on a Chromium based browser:
+After running the above commands,
+1. Navigate to `chrome://extensions`. 
+2. Enable the `Developer Mode`.
+3. Select `Load Extension` and choose the generated `dist` directory from the project folder to install it.
+
+You should now have the extension installed and running.
+
+## 🙌🏼 Core Team
+* [Brian Douglas](https://github.com/bdougie)
```

Comment:
Do you think maybe something like this would look better - https://github.com/openfoodfacts/taxonomy-editor#contributors?